Tests/PackageLoadingTests/ConventionTests.swift

```diff
@@ -463,16 +463,23 @@ class ConventionTests: XCTestCase {
         let fs = InMemoryFileSystem(emptyFiles:
             "/Sources/Foo/Foo.swift",
             "/Sources/Bar/Bar.swift",
-            "/Sources/Baz/Baz.swift")
+            "/Sources/Baz/Baz.swift",
+            "/Sources/Exec👽/main.swift"
+        )
```

Comment:
Did you also make sure that the Build module won't do anything stupid with this (like try to link it in and fail)? We perhaps should have test coverage of that too.